packages/react/src/themes/teams/components/Checkbox/checkboxVariables.ts

```diff
@@ -1,22 +1,75 @@
+import { pxToRem } from '../../../../lib'
 
-export type CheckboxVariables = Partial<BaseCheckboxVariables>
+export interface CheckboxVariables {
+  rootPadding: string
+
+  checkboxTextColor: string
+  checkboxBackground: string
+  checkboxBorderColor: string
+  checkboxBorderStyle: string
+  checkboxBorderRadius: string
+  checkboxBorderWidth: string
+  checkboxIndicatorColor: string
+  checkboxGap: string
+  checkboxMargin: string
+  checkboxPadding: string
+
+  checkboxTextColorHover: string
+  checkboxBorderColorHover: string
+  checkboxCheckedBackgroundHover: string
+  checkboxCheckedTextColor: string
+  checkboxCheckedBackground: string
+  checkboxCheckedBorderColor: string
+  checkboxCheckedIndicatorColor: string
+
+  toggleBorderRadius: string
+  toggleIndicatorColor: string
+  toggleMargin: string
+  togglePadding: string
+  toggleCheckedPadding: string
+
+  disabledColor: string
+  disabledCheckboxBackground: string
+  disabledCheckboxBackgroundChecked: string
+  disabledCheckboxBorderColor: string
+  disabledCheckboxCheckedIndicatorColor: string
+  disabledToggleIndicatorColor: string
+}
+
+const [toggleMovementDistance, checkboxPadding] = [20, 2].map(v => pxToRem(v))
 
 export default (siteVars: any): CheckboxVariables => ({
-  checkboxBorderColor: siteVars.colors.grey[750],
-
-  toggleColor: siteVars.colors.brand[600],
-
-  checkedCheckboxBackground: siteVars.colors.brand[600],
-  checkedCheckboxBorderColor: siteVars.colors.brand[600],
-  checkedCheckboxColor: siteVars.colors.white,
-  checkedToggleBackground: siteVars.colors.brand[600],
-  checkedToggleBorderColor: siteVars.colors.brand[600],
-  checkedToggleColor: siteVars.colors.white,
-
-  disabledColor: siteVars.colors.grey[250],
-  disabledCheckboxBackground: siteVars.colors.grey[250],
-  disabledCheckboxBorderColor: siteVars.colors.grey[250],
-  disabledToggleBackground: siteVars.colors.grey[250],
-  disabledToggleBorderColor: siteVars.colors.grey[250],
+  checkboxTextColor: siteVars.colorScheme.default.foreground1,
+  checkboxBackground: siteVars.colorScheme.default.background,
+  checkboxBorderColor: siteVars.colorScheme.default.foreground1,
+  checkboxBorderStyle: 'solid',
+  checkboxBorderRadius: pxToRem(3),
+  checkboxBorderWidth: pxToRem(1),
+  checkboxIndicatorColor: 'transparent',
+  checkboxGap: pxToRem(12),
+  checkboxMargin: `${pxToRem(20)}, 0`,
+  checkboxPadding,
+  rootPadding: '3px 5px',
+
+  checkboxTextColorHover: siteVars.colorScheme.default.foreground,
+  checkboxBorderColorHover: siteVars.colorScheme.default.foreground,
+  checkboxCheckedBackgroundHover: siteVars.colors.brand[600],
+
+  toggleBorderRadius: pxToRem(999),
+  toggleIndicatorColor: siteVars.colors.brand[600],
+  toggleMargin: `${pxToRem(20)}, 0`,
+  togglePadding: `${checkboxPadding} ${toggleMovementDistance} ${checkboxPadding} ${checkboxPadding}`,
+
+  checkboxCheckedTextColor: siteVars.colorScheme.default.foreground,
+  checkboxCheckedBackground: siteVars.colors.brand[600],
```

Comment:
I *think* this should be `siteVars.colorScheme.brand.backgroundActive1` 
![image](https://user-images.githubusercontent.com/828692/61326582-f610b300-a7cb-11e9-8cf5-ac68f493fd23.png)

This will also get rid of the need for defining `checkboxCheckedBackground` in the `teams-high-contrast` theme file.